source/intel.js

```diff
@@ -1977,12 +1977,59 @@ NeptunesPride.npui.StarInspector = function () {
 
   return starInspector;
 };
+//Enable arrows keys to interact with editing a ship's instructions on stay.
+const fleet_order_keyboard_integeration = () => {
+  const event_name = "show_screen";
+
+  const selector_right_button =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_black > div:nth-child(3)";
+  const selector_left_button =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_black > div:nth-child(2)";
+  const selector_menu =
+    "#contentArea > div > div.widget.fullscreen > div:nth-child(3) > div > div.widget.rel.col_accent > div.widget.drop_down > select";
+
+  const apply_arrow_click = (selector, key) => {
+    console.log("Event added");
+    const NP_click = (event) => {
+      if (event.keyCode === key) {
```

Comment:
This code is technically correct but in modern browsers I think using `event.key` and the key names is probably preferable. Equivalent code is

```
if (event.key === key) { 
```

where instead of `key` being a number it is one of `"ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown"` (among other choices, these are just the names you might want here).

Reply:
Yes, could you please make a commit changing that plz? I don't mind and would prefer the modern approach.

Reply:
I see you made a commit of this difference